lab-daniel/src/main.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const fileReader = require('./lib/reader');
+
+const baconPath = `${__dirname}/data/bacon.txt`;
+const hodorPath = `${__dirname}/data/hodor.txt`;
+const bobRossPath = `${__dirname}/data/bob-ross.txt`;
+
+const printCharacters = (file) => {
+  console.log(file);
```

Comment:
Eeeek! So much text! Methods such as `String.prototype.slice` or `String.prototype.substring` will help you out a lot here. :)